website/content/docs/job-specification/resources.mdx

```diff
@@ -86,5 +86,49 @@ resources {
   }
 }
 ```
+## Memory Oversubscription
+
+Setting task memory limits is a challenging act of balance. Job authors must set
+a memory limit for each task — if it’s too low, the task may be interrupted if
+it exceeds the limit unexpectedly; if it’s too high, the cluster is left
+underutilized and resources are wasted. Job authors usually set limits based on
+their task's typical memory usage and then add a safety margin to handle
+unexpected load spikes or uncommon scenarios. That means that in some clusters,
+20-40% of the cluster memory may be reserved but unused.
+
+To help minimize reserved but not used cluster memory, Nomad 1.1 lets job
+authors set two separate memory limits:
+
+* `memory`: the reserve limit to represent the task’s typical memory usage —
+  this number is used by the Nomad scheduler to reserve and place the task
+
+* `memory_max`: the maximum memory the task may use, if the client has excess
+  available memory, and may be terminated if it exceeds
+
+If a client's memory becomes contended or low, the operating system will
+pressure the running tasks to free up memory. If the contention persist, Nomad
+may kill oversubscriped tasks and reschedule them to other clients. The exact
+mechanism for memory pressure is specific to the task driver and application
+runtime.
+
+The new max limit attribute is currently supported by Docker and all default
+Nomad drivers except raw_exec and QEMU. Consult the third-party drivers
+documentation for memory oversubscription support.
```

Comment:
Let's make the supported driver list explicit. Making the list explicit also makes me ask if `podman` has support? It's neither "default" nor "third-party"

```suggestion
The new max limit attribute is currently supported by the official `docker`,
`exec`, and `java` task drivers.  Consult the documentation of
community-supported task drivers for their memory oversubscription support.
```